# timestamp, frame id
Header header

#set of targets being tracked
person[] peopleSet